source/plugin/Assets/GoogleMobileAds/Platforms/Unity/BannerClient

```diff
@@ -0,0 +1,238 @@
+
+            if (adSize == AdSize.SmartBanner)
+            {
+                AdBehaviour.SetAndStretchAd(prefabAd, 0);
+                rect.anchoredPosition = new Vector3(0, y, 1);
+            }
+            else
+            {
+                rect.anchoredPosition = new Vector3(x, y, 1);
+            }
+            
+        }
+
+        // Requests a new ad for the banner view.
+        public void LoadAd(AdRequest request)
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            ShowBannerView();
+            
+        }
+
+        // Shows the banner view on the screen.
+        public void ShowBannerView()
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            
+            Image myImage = prefabAd.GetComponentInChildren<Image>();
+            RectTransform rect = myImage.GetComponent<RectTransform>();
+            dummyAd = AdBehaviour.ShowAd(prefabAd, rect.anchoredPosition);
+            
+        }
+
+        // Hides the banner view from the screen.
+        public void HideBannerView()
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            
+            AdBehaviour.DestroyAd(dummyAd);
+        }
+
+        // Destroys a banner view.
+        public void DestroyBannerView()
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            if (dummyAd != null)
+            {
+                AdBehaviour.DestroyAd(dummyAd);
+            }
+        }
+
+        // Returns the height of the BannerView in pixels.
+        public float GetHeightInPixels()
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            return 0;
+        }
+
+        // Returns the width of the BannerView in pixels.
+        public float GetWidthInPixels()
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            return 0;
+        }
+
+        // Set the position of the banner view using standard position.
+        public void SetPosition(AdPosition adPosition)
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+        }
+
+        // Set the position of the banner view using custom position.
+        public void SetPosition(int x, int y)
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+        }
+
+        // Returns the mediation adapter class name.
+        public string MediationAdapterClassName()
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            return null;
+        }
+
+        // Returns ad request Response info client.
+        public IResponseInfoClient GetResponseInfoClient()
+        {
+            Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            return null;
+        }
+
+        //Event Callbacks
+        public void onAdLoaded()
```

Comment:
this is android specific so won't do anything. None of these callbacks have any effect here, I would remove.